charts/ccsm-helm/README.md

```diff
@@ -238,9 +238,47 @@ Information about Tasklist you can find [here](https://docs.camunda.io/docs/comp
 | | `ingress.enabled` | If true, an ingress resource is deployed with the Tasklist deployment. Only useful if an ingress controller is available, like nginx. | `false` |
 | | `ingress.className` | Defines the class or configuration of ingress which should be used by the controller | `nginx` |
 | | `ingress.annotations` | Defines the ingress related annotations, consumed mostly by the ingress controller | `ingress.kubernetes.io/rewrite-target: "/"` <br/> `nginx.ingress.kubernetes.io/ssl-redirect: "false"` |
-| | `ingress.path` | Defines the path which is associated with the Tasklist service and port https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules | `/` |
+| | `ingress.path` | Defines the path which is associated with the Tasklist [service and port](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules) | `/` |
 | | `ingress.host` | Can be used to define the [host of the ingress rule.](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules) If not specified the rules applies to all inbound HTTP traffic, if specified the rule applies to that host. | `""` |
 
+### Identity
+
+Information about Identity you can find [here](https://docs.camunda.io/docs/self-managed/identity/what-is-identity/).
+
+| Section | Parameter | Description | Default |
+|-|-|-|-|
+| `identity`| |  Configuration for the identity sub chart. | |
+| | `enabled` |  If true, the identity deployment and its related resources are deployed via a helm release | `true` |
+| | `image` |  Configuration to configure the identity image specifics | |
+| | `image.repository` |  Defines which image repository to use | `camunda/identity` |
+| | `image.tag` |  Can be set to overwrite the global tag, which should be used in that chart | |
```

Comment:
Actually this is optional since the global tag is always set, which defines for all services the version (normally they have all the same version :D )

Reply:
Yeah, that part was clear to me :+1: I found the original wording a bit confusing because I read it as "the global tag which should be used in that chart" which didn't make sense to me.